src/components/ChatEntry.js

```diff
@@ -1,22 +1,36 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateLike, color}) => {
+
+  const likedButtonContent = liked ? '❤️' : '🤍';
+  const localOrRemote = (sender === 'Vladimir') ? 'chat-entry local' : 'chat-entry remote';
+  
 
-const ChatEntry = (props) => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+    <div className={localOrRemote}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p className={color}>{body}</p>
+          <p className="entry-time">
+              <TimeStamp time={timeStamp} /></p>
```

Comment:
[nit] It helps readability if siblings are at the same level of indentation. I thought the second `<p` was inside the first, at first. :)